apps/condo/domains/notification/adapters/redStore/redStoreNotificationSender.js

```diff
@@ -0,0 +1,69 @@
+const { get } = require('lodash')
+
+
+const { fetch } = require('@open-condo/keystone/fetch')
+
+const { APS_RESPONSE_STATUS_SUCCESS } = require('../apple/constants')
+
+
+class RedStoreNotificationSender {
+    _projectId = null
+    _serviceToken = null
+    _url = null
+
+    /**
+     * @param config
+     */
+    constructor (config) {
+        this._projectId = config['project_id']
+        this._serviceToken = config['service_token']
+        this._url = config['url']
+        this.sendPush = this.sendPush.bind(this)
```

Comment:
why are you doing this? this.sendPush by default going to work in this execution context. In case if you would like to use this function outside this execution context, but still bind to this object - you have to override contexts in the point of reusing function